cmd/gapir/CMakeBuild.cmake

```diff
@@ -31,6 +31,13 @@ if(NOT DISABLED_CXX)
     elseif(NOT GAPII_TARGET)
         add_executable(gapir ${sources})
         install(TARGETS gapir DESTINATION ${TARGET_INSTALL_PATH})
+        if(MSVC)
+            add_custom_command(
```

Comment:
Would this be better as an actual target? I'm not a big fan of the "/../../../../" path - reading this, I have no idea where it goes.

Reply:
I agree, but I wasn't sure how to specify the gapir-msvc target directory from the custom command. Do you think it would work if I made it a ExternalProject_Step that depended on gapir?